### PR TITLE
Seeding bidders for Bureau bids

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
🤮  i've never done sql stuff before 👀 

Thanks for the pseudocode Scott!

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
Trying to add bids for all `perdets` for all of the `cpids` that belong to the bureaus that come through for Wendy and Admin.

File: seeds/016_bids_seed.js [hunk 1]
```diff
@@ -19,9 +24,15 @@ exports.seed = function(knex) {
                       cp_id: findRandom(ap)['cp_id'],
                     })
                   }
+                  bureauPositions().forEach(cpid => {
```
Comment (elizabeth-jimenez) [inline seeds/016_bids_seed.js:27]: have to move this out, so that it can use the results from line 14-16.

**Comment by elizabeth-jimenez** [comment]
Couldn't have done it without @scott062 ! many thanks!

**Review by scott062 — APPROVED** [review]
👍

**Pull request merged**